shared/js/ui/views/grade-scorecard.es6.js

```diff
@@ -0,0 +1,90 @@
+const Parent = require('./sliding-subview.es6.js')
+
+function GradeScorecard (ops) {
+  this.model = ops.model
+  this.template = ops.template
+
+  Parent.call(this, ops)
+
+  this.setupClose()
+}
+
+GradeScorecard.prototype = window.$.extend({},
+  Parent.prototype,
+  {
+    _render: function (ops) {
+      // doing this so the template can get access to the reasons
+      // TODO put them on a model? allow them to be passed via the arguments
+      // to _render like we do in core?
+      this.reasons = this._getReasons()
```

Comment:
Looking at this again, it's probably best to move all this to a model. I'll do that next week.